src/ert/shared/hook_implementations/workflows/disable_parameters.py

```diff
@@ -3,18 +3,57 @@
 
 class DisableParametersUpdate(ErtScript):
     """The DISABLE_PARAMETERS workflow disables parameters,
-    so they are excluded from the update step. The job takes a list
+    so they are excluded from the update step. The job takes a group
     of parameters as input:
 
-    DISABLE_PARAMETERS "PARAMETER_1, PARAMETER_2"
+    DISABLE_PARAMETERS PRED
 
     The parameters that are given as arguments will be removed from the
-    update. Note that if giving more than one parameter as input the list
-    must be enclosed in quotes.
+    update.
 
     Note that unknown parameter names will be silently ignored.
 
-    This workflow is recommended to be run as a PRE_FIRST_UPDATE hook
+    This workflow is recommended to be run as a PRE_FIRST_UPDATE hook.
+
+    As example to disable parameters in the poly.ert case (located in
+    ert/test-data/poly_example) has three steps:
+
+    First, in the ert config file ("poly.ert"), include:
+
+        LOAD_WORKFLOW disable_parameter_pred
+        HOOK_WORKFLOW disable_parameter_pred PRE_FIRST_UPDATE
+
+        GEN_KW PRED coeff.tmpl coeffs_pred.json coeff_priors
+        GEN_KW COEFFS coeff.tmpl coeffs.json coeff_priors
+
+    Note that the output file from the groups should be different to avoid overwriting
+    values sampled.
+
+    Second, make a new file called "disable_parameter_pred" with:
+
+        DISABLE_PARAMETERS PRED
+
+    Finally, analyse the plots. There would be two groups of parameters:
+
+        COEFFS:COEFF_A
+        COEFFS:COEFF_B
+        COEFFS:COEFF_C
+
+        PRED:COEFF_A
+        PRED:COEFF_B
+        PRED:COEFF_C
```

Comment:
I think it would be better to just remove all the references to the poly example, and just show the two lines of `GEN_KW`, and make sure that the file names are different, instead of writing that in text, as that it technically not related to configuring this. For example:

```
GEN_KW PRED pred.tmpl pred.json pred_priors
GEN_KW COEFFS coeff.tmpl coeffs.json coeff_priors
```
And if you would like to reference `COEFF_A` and so forth it is probably worth showing the content of for example `coeff_priors` to show where they are coming from.

Reply:
Good point, then it seems to be better to rename the parameters in "pred_priors" as well. Will try with COEFF_D ... to get your review if this is correct.